sunpy/map/tests/test_mapbase.py

```diff
@@ -879,17 +883,14 @@ def test_rotate(aia171_test_map):
     np.testing.assert_allclose(rotated_map_2.data[0, 0], 0., atol=1e-7)
     assert rotated_map_2.mean() < rotated_map_1.mean() < aia171_test_map.mean()
 
-    rotated_map_3 = aia171_test_map.rotate(0 * u.deg, scale=1.5)
-    assert rotated_map_3.mean() > aia171_test_map.mean()
+    # A scaled-up map should have the same mean because the output map should be expanded
+    rotated_map_3 = aia171_test_map.rotate(0 * u.deg, order=3, scale=2)
+    np.testing.assert_allclose(aia171_test_map.mean(), rotated_map_3.mean(), rtol=1e-4)
 
-    # Mean and std should be equal when angle of rotation is integral multiple
-    # of 90 degrees for a square map
-    rotated_map_4 = aia171_test_map.rotate(90 * u.deg, scale=1.5)
-    np.testing.assert_allclose(rotated_map_3.mean(), rotated_map_4.mean(), rtol=1e-3)
-    np.testing.assert_allclose(rotated_map_3.std(), rotated_map_4.std(), rtol=1e-3)
-    rotated_map_5 = aia171_test_map.rotate(180 * u.deg, scale=1.5)
```

Comment:
Why get rid of the 180deg test? Is it redundant with the other tests?

Reply:
Yes, it should be redundant with the other tests unless there's a pathological problem with the rotation implementation (which shouldn't be the responsibility of this test to catch).  You may note that I changed the order of the 90-degree rotation so that I could severely tighten the test tolerance.  I can't do the same for the 180-degree test because rounding actually causes the width to be padded out.  Rather than mucking around with cropping the rotated map back down, I opted instead to remove the test.